build/Makefile

```diff
@@ -196,6 +204,12 @@ ifeq ($(ULINUX),1)
 endif
 
 #--------------------------------------------------------------------------------
+# Build AUOMS project
+
+$(AUOMS_TARGET_DIR)/auoms-bundle-test.sh : 
```

Comment:
auoms-bundle-test.sh is not the bundle. It is the script that checks to see if the bundle can be installed. It serves the same purpose as the *-oss-test.sh scripts.